src/FileCacheManager.php

```diff
@@ -49,6 +50,7 @@ public function __construct($isEnabled, $cacheFile, array $rules)
     {
         $this->isEnabled = $isEnabled;
         $this->cacheFile = $cacheFile;
+        $this->cacheFileRealDirName = dirname(realpath($cacheFile));
```

Comment:
OK, I think I may know what was in your mind.
calling realpath on phar:// stream won't provide useful data, but calling it (even inside phar file) on file that is outside phar (like here) is ok